src/Compilers/Core/CodeAnalysisTest/Analyzers/AnalyzerFileReferenceAppDomainTests.cs

```diff
@@ -94,9 +94,9 @@ public class TestAnalyzer : DiagnosticAnalyzer
                 new SyntaxTree[] { CSharp.SyntaxFactory.ParseSyntaxTree(analyzerSource) },
                 new MetadataReference[]
                 {
-                    NetStandard20.mscorlib,
-                    NetStandard20.netstandard,
-                    NetStandard20.SystemRuntime,
+                    NetStandard20.References.mscorlib,
```

Comment:
nit: was looking at the code structure in the basic-reference-assemblies repo (for [example](https://github.com/jaredpar/basic-reference-assemblies/blob/main/Basic.Reference.Assemblies.NetStandard20/Generated.cs#L493)), consider putting some xml doc comments on `References` and `ReferenceInfos`.

Reply:
Yeah looking at that code isn't going to be particularly useful. It's 99% just generated code. 

XML docs are a good idea though. Will add those

Reply:
https://github.com/jaredpar/basic-reference-assemblies/commit/5eeecbf9127aee17b960b1d84a233218e213a420

Those docs will show up next upgrade of these packages